app/javascript/test/setup.js

```diff
@@ -2,9 +2,37 @@ import Vue from 'vue'
 import _ from 'lodash'
 import VueDetails from 'vue-details'
 import system from 'lux-design-system'
+import vueSnapshotSerializer from 'jest-serializer-vue'
 
 Vue.use(system)
 Vue.config.productionTip = false
 Vue.component('v-details', VueDetails)
-jest.unmock('lodash')
-_.debounce = jest.fn((fn) => fn);
+vi.unmock('lodash')
+_.debounce = vi.fn((fn) => fn);
+expect.addSnapshotSerializer(vueSnapshotSerializer)
+
+const oldWindowLocation = window.location
+
+beforeAll(() => {
```

Comment:
Add a comment explaining what this is for

Reply:
Done.